sklearn/linear_model/tests/test_ridge.py

```diff
@@ -711,3 +712,45 @@ def test_ridge_fit_intercept_sparse():
     assert_warns(UserWarning, sparse.fit, X_csr, y)
     assert_almost_equal(dense.intercept_, sparse.intercept_)
     assert_array_almost_equal(dense.coef_, sparse.coef_)
+
+
+def test_errors_and_values_helper():
+    ridgecv = _RidgeGCV()
+    rng = check_random_state(42)
+    alpha = 1.
+    n = 5
+    y = rng.randn(n)
+    v = rng.randn(n)
+    Q = rng.randn(len(v), len(v))
```

Comment:
I'd much prefer if we had better variable names. Currently, I have a hard time understanding this example, and that's a problem.